hpcflow/sdk/core/workflow.py

```diff
@@ -886,7 +886,7 @@ def url(self) -> str:
         if self._store.fs:
             if self._store.fs.protocol == "zip":
                 return self._store.fs.of.path
-            elif self._store.fs.protocol == "file":
+            elif self._store.fs.protocol == ("file", "local"):
```

Comment:
Do you mean `...protocol in ("file", "local")`?

Reply:
No, I think it's correct like this: https://github.com/fsspec/filesystem_spec/blob/aa1a301f13d086c9c664c61f3838911418f52905/fsspec/implementations/local.py#L30